tests/integration-tests/block-handlers/truffle.js

```diff
@@ -1,6 +1,19 @@
 require("babel-register");
 require("babel-polyfill");
 
+const Web3 = require("web3");
+const HDWalletProvider = require("@truffle/hdwallet-provider");
+
+Web3.providers.HttpProvider.prototype.sendAsync = Web3.providers.HttpProvider.prototype.send
+const neonDevnet = "https://devnet.neonevm.org"
+const provider = new Web3.providers.HttpProvider(neonDevnet);
+
+const privateKeys = [
+  '0x7efe7d68906dd6fb3487f411aafb8e558863bf1d2f60372a47186d151eae625a',
```

Comment:
why we are using this keys in public mode? better use it from secrets

Reply:
It was accidentally commited, I will fix. Of course I get it from env